### ISSUE TITLE
[Feature Request] alpine 3.18 package is missing

### ISSUE THREAD

**Issue by giulianozor** [zerotier/ZeroTierOne#2020]
Please,
Can you add the package to alpine 3.18 repository ?

Thank you!

**Comment by el-vishal** [comment]
your commands helped me! Thank you for sharing
+1 on this request

**Comment by hubrichc** [comment]
thank you for sharing

**Comment by ShaRose** [comment]
Just to add since I did some digging: The cause was that alpine ports need to be an open source license, which zerotier isn't.

Not really a fix for this unless zerotier hosts a proper alpine repo,

https://git.alpinelinux.org/aports/commit/community?id=7f7b355d081655ed3fbbc6391f6aac987d2d4e2c

https://wiki.alpinelinux.org/wiki/Package_policies

"The license of the program should be based on the licenses approved by the [Free Software Foundation](http://www.gnu.org/philosophy/license-list.html#GPLCompatibleLicenses) or [OSI](http://www.opensource.org/licenses/)."

Seems whoever initially submitted the package didn't look closely and put it in as 'GPL-3.0-or-later', when it should have been 'BUSL-1.1'.

**Comment by donluca** [comment]
Seconding this.

Even if we can't have ZeroTier in the Alpine Linux package manager, it would be nice if the developers could provide standalone packages like they do for Debian/RPM.

**Comment by kormose** [comment]
Still missing in Alpine 3.19.

**Comment by underwoodblog** [comment]
...and 3.20

**Comment by NepBreadus** [comment]
+1 no ZeroTier for 3.20.

**Comment by kormose** [comment]
Packages or build instructions would be great to have for Alpine.

**Comment by BloodyIron** [comment]
Yeah I don't think this is going to happen folks. It's clear that there's a license conflict here, so for the areas I care about this, I'mm just going to migrate away from Alpine Linux and towards kubernetes implementations with the already-available docker images. That way I don't have to deal with the junk that is Alpine Linux (ugh their default logging enrages me so much).

**Comment by BloodyIron** [comment]
I'm just going to move away from Alpine Linux, I'm having more and more problems with it.

**Comment by llbranco** [comment]
following this guide to install on Alpine 3.22 kernel 6.12.33 standard x86/64

https://blog.florianuhlemann.de/compile-zerotier-one-on-alpine-linux-from-github-source/
